providers/google/src/airflow/providers/google/cloud/hooks/gcs.py

```diff
@@ -726,6 +726,14 @@ def delete(self, bucket_name: str, object_name: str) -> None:
 
         self.log.info("Blob %s deleted.", object_name)
 
+    def get_bucket(self, bucket_name: str) -> storage.Bucket:
```

Comment:
This method makes sense. But when I check the rest of this file, I realize that we are using the following snippet repeatedly:
```python
client = self.get_conn()
client.bucket(bucket_name)
```
I also see that some places, we are using the `client.bucket` like the following
```python
client.bucket(bucket_name, user_project=user_project)
```
where `user_project` is a parameter of the function:
> :param user_project: The identifier of the Google Cloud project to bill for the request. Required for Requester Pays buckets.

Do you think that we can add this optional parameter (`user_project`) to this function?

Reply:
My PR resolves extremely important feature from SFTPToGCSOperator.
Currently you cannot copy big files (10GB for example)
Initially I wanted to use native client methods without modifying `gcs.py`, @potiuk asked me to use abstracted hook.

My focus is to improve SFTPToGCS. I don't have time to perform refactor of `gcs.py` file, as I'm already involved in next project